iTunesSearch/ItunesSearch.m

```diff
@@ -96,6 +97,10 @@ - (void)performApiCallForMethod:(NSString*)method
     if (self.affiliateToken && self.affiliateToken.length > 0)
         newParams[@"at"] = self.affiliateToken;
 
+    // Add campaign token if supplied
+    if (self.campaignToken && self.campaignToken.length > 0)
```

Comment:
I could absolutely do that, but don't you want me to follow the code style? See the lines just before... Just checking :)